packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -307,6 +307,15 @@ export class Outbox {
 		return this.params.opReentrancy() && !this.rebasing;
 	}
 
+	/**
+	 * As necessary and enabled, compresses and chunks the given batch.
+	 *
+	 * @remarks - If chunking happens, a side effect here is that 1 or more chunks are queued immediately for sending in next JS turn.
```

Comment:
This is technically true, but maybe worth calling out that the "waiting for next JS turn" in this case happens at the loader layer (ConnectionManager's `outbound` queue).
This also applies to every other instance of sending messages up to the loader layer.